.devcontainer/docker-compose.yaml

```diff
@@ -6,8 +6,8 @@ services:
     build:
       context: ../v2g-liberty
       dockerfile: Dockerfile.dev
-    ports:
-      - 5050:5050
+    # ports:
```

Comment:
This actually works for the devcontainer. 
Is there any function that could be useful?